packages/validator/src/util/clock.ts

```diff
@@ -138,7 +138,7 @@ export class Clock implements IClock {
  */
 export function getCurrentSlotAround(config: ChainForkConfig, genesisTime: TimeSeconds): Slot {
   const diffInSeconds = Date.now() / 1000 - genesisTime;
-  const slotsSinceGenesis = Math.floor(diffInSeconds / config.SECONDS_PER_SLOT);
+  const slotsSinceGenesis = Math.round(diffInSeconds / config.SECONDS_PER_SLOT);
```

Comment:
The tests data we have for this function suggests that for 12s slot window clock at 12.5 would still be `zero` slot. To align implementation with the tests data we have to use `Math.floor` here. Or else if that's not the case then we should change the tests data else it will be fail.

Reply:
Yes the answer is to fix the test cases, not force the implementation to change for the buggy tests.
Merging this for now to fix the noticeable validator performance regression.

Reply:
Yes test cases don't enforce implementation, but fixture data particularly guide what kind of behavior we expect form the implementation.